src/frontend/src/lib/components/ui/ButtonMenu.svelte

```diff
@@ -1,11 +1,13 @@
 <script lang="ts">
+	export let disabled = false;
 	export let ariaLabel: string;
 </script>
 
 <button
 	class="no-underline hover:text-blue active:text-blue w-full text-left"
 	aria-label={ariaLabel}
 	on:click
+	{disabled}
```

Comment:
Noticing it know, I think on button, if disabled, we also tweak the opacity to make this visually recognizable, like `class:opacity-50={disabled}` (I think). Maybe something worth adding here too?

Reply:
right! i forgot about it, tks!